### DISCUSSION TITLE
Update containers that are installed/spin up with docker run -d

### DISCUSSION THREAD

**Discussion by jicho** [louislam/dockge#193] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Make it possible to update running containers that are installed using `docker run -d [etc]` 

### ✔️ Solution

It would be nice to make it possible to use the "update" function/button in a same way Portainer does:

<img width="471" alt="image" src="https://github.com/louislam/dockge/assets/610740/fa0f4bbb-a8a5-4742-872a-fb188edc408f">

This would mean that users can use dockge for docker compose maintenance, but also containers that were started using `docker run`. That means Portainer isn't needed anymore and dockge can be a single point of maintenance system :)

### ❓ Alternatives

Stay with Portainer for updating containers that are not "installed" using the docker compose option.

**Comment by louislam** [comment]
I think you are looking into a wrong project. Dockge is focusing on stack management only. It is not a Portainer replacement.

In the home page, there a text field for converting your `docker run` command to a compose file.

FAQ:
> Can I manage a single container without compose.yaml?

> The main objective of Dockge is to try to use the docker compose.yaml for everything. If you want to manage a single container, you can just use Portainer or Docker CLI.

**Comment by jicho** [comment]
Argh, sorry... missed that in the FAQ :S
